project_milestone/models/project.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class Project(models.Model):
+    _inherit = 'project.project'
+
+    milestone_ids = fields.One2many('project.milestone',
+                                    'project_id',
+                                    string="Milestones")
+    use_milestones = fields.Boolean(string="Use milestones")

```

Comment:
```suggestion
    use_milestones = fields.Boolean()
```

It will be the same.